### DISCUSSION TITLE
Loaded 1.2.2 First Time, BLT Doesn't Touch Bed

### DISCUSSION THREAD

**Discussion by AhJuicyKiwi** [Jyers/Marlin#379] [Q&A] [answered]
So after loading this new firmware since I've only heard good things in groups I'm part of. So afterwards I calibrated e-steps changed that settings and saved. Then I moved on to auto home, well it'll locate in X and Y like normal BUT it'll move POSITIVE in Z then state "Stopped" if I do Auto Level it'll do the same as homing but stop and say completed do I want to save to EPPROM but never touches the bed..

So am I missing a crucial step??? Thought this would be easy haha please and thank you to everyone in advance!!

AhJuicyKiwi

**Comment by okcub2010** [comment]
I just installed a 427 board and v2 display and reused my b2.0 a blt  on my older e3 pro so like you I’m a first time jyers Fw user.
You haven’t mentioned the board version fw your using?
In my case I’ve tried the 427 blt HS 1.2.2 3x3 and 5x5 versions. I had to set the probe X offset in the Advanced menu to -50 to have the probe not go off the left edge.
Auto home worked ok for me.
To set z offset I did a first layer test print with a g29 in the slicer start code after the g28.
My blt did probe the bed but started printing high off the bed. I then disked in z offset while printing and found in my case that a -2.95 setting laid down filament to my liking.
that value was saved and no real issues with the 3x3 or 5x5 HS 427 fw except it doesn’t probe fast as indicated? The normal and HS versions deep to probe at the same speed. 
Im looking itto g this but I’m sure it’ll be solved I’ve posted a request to look into it.
So I hope this helps a little knowing it should work.
More follow up things you’ve tried and fw and board used csn help.

**Reply by AhJuicyKiwi** [reply]
I currently have a 4.2.2 board with 1.2.2 jeers loaded. I'm still confused why I'm not able to move Z axis though at all. Is there anything specific needed before you download firmware? I originally loaded Creality firmware then just flashed to Jyers. 

Thanks for the help man

**Reply by okcub2010** [reply]
I only tried the 3c4 and 5x5 HS BL touch fw for the 427 board but I was able to select X the Y then Z and move them using the dial.I had to heat the extruder up to 180c and then I could move E.
If you flashed the 422 fw version it should work. Just go thru the menus?

**Comment by Jyers** [comment, marked as answer]
Are you still facing this issue? Sounds like it could possibly be a flakey connection, as I see that a lot with weird probe behavior.

**Reply by okcub2010** [reply]
I bought and installed a newer v3.1 BL touch and it's fast probing correctly now. It seems one of my v3.0 will fast probe that's newer early 2019) and one won't that's a year older (late 2019) just before being replaced by the v3.1 version.
All's ok now.

**Reply by AhJuicyKiwi** [reply]
Yes its still going on... I've tried different probe settings and pin settings in the config but I was able to get everything moving appropriately but now after homing the probe doesn't deploy and the head keeps going down in Z as if it'll go through the bed... So I'm definitely lost now and I used the newest firmware you released as well

**Comment by normanu** [comment]
I actually still have this issue.
flashed E3V2-BLTouch-5x5-v4.2.2 and X and Y home ok, but Z stays way high up.
With move it will move upward, but won't move down......
Any idea?

It worked with a previous smith3d fw

**Reply by Jyers** [reply]
My best guess would be a connection issue. Is your BL Touch connected to the Z Endstop or the dediacted BL Touch port?

**Reply by normanu** [reply]
connected to the z endstop.

**Reply by Jyers** [reply]
For this firmware you need to be connected to the BL Touch port. If that is an issue you can always compile your own for use with the endstop, but switching the connector is easy enough if that is possible for you.

**Reply by AhJuicyKiwi** [reply]
> For this firmware you need to be connected to the BL Touch port. If that is an issue you can always compile your own for use with the endstop, but switching the connector is easy enough if that is possible for you.

So when I installed my BLtouch I removed the z stop pin and connected the BL pin to that. But even changing the config I can't seem to get it to deploy now but all the axis are homed correctly besides Z to bed

**Reply by AhJuicyKiwi** [reply]
> For this firmware you need to be connected to the BL Touch port. If that is an issue you can always compile your own for use with the endstop, but switching the connector is easy enough if that is possible for you.

The BLT I have installed uses a red Z pin connector and a adapter split into the display port on the board. I've tried everything in the source code to try and fix the probe issue but nothing has worked..

**Reply by Jyers** [reply]
Have you tried uncommenting `#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN`

**Reply by wszamko** [reply]
I had the same problem as my BL Touch 3.1 was connected using the board, not directly to the BL Touch port on the 4.2.7. I removed the board and connected using this diagram (https://i0.wp.com/www.smith3d.com/wp-content/uploads/2020/08/7.jpg?w=1000&ssl=1) and it worked perfectly.

**Comment by normanu** [comment]
Ah ok!
Z switch still needs to be disconnected?
The change from the previous version to this new was not obvious on the
Smith3D guide.
Thanks!!

Op do 8 apr. 2021 23:49 schreef Jyers ***@***.***>:

**Reply by Jyers** [reply]
You don't have to disconnect the z switch, but there's no real reason to have it connected.
Sorry for the confusion, I have no real affiliation with Smith3D apart from them deciding to recommend my firmware. I'll see if I can contact them to recommend that they add that to the guide.

**Comment by okcub2010** [comment]
I've not had any problems like that so I ask done questions. 
Does the BL touch at power up glow red and drop and then retract the probe?
If you with power off rotate z go raise the x axis high off the bed and then do a auto home that should drop the probe and move towards the bed,if you let the probe hit your finger does the z movement stop ?
This tests the BL touch is working and the probe is controlling the z stop. If yes it's not a BL touch problem.
It would also indicate your BL touch connections are ok.
I have to ask, you did unplug or remove the z microswitch? 
You are using the 5 pin board BL Touch connector?

**Comment by okcub2010** [comment]
This helped me a lot.
Just substitute Jyers for Smith3D fw.
My cable has the red 2 pin so I just cut the tab off abs it fit the 5 pin connector better. All my blt connectors are using the 5 pin connector. No reason to use the older pin27 board because it interrupts the buzzer wire that's not needed with the new fw abs 5 pin connector. I used hot glue to secure the blt pins in the 5 pin. Very important that they don't come loose.
Dr Vax blt install video.
Board wiring and fw flashing and setup works for 422 and 427 boards installed in e3,e3 pro and v2 printers regardless of the hotend mount as long as it's in the same location. 
  
   __ https://youtu.be/jO4PAWvWdH8
